src/Microsoft.FeatureManagement.Telemetry.ApplicationInsights/TargetingTelemetryInitializer.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.DataContracts;
+using Microsoft.ApplicationInsights.Extensibility;
+using System.Diagnostics;
+
+namespace Microsoft.FeatureManagement.Telemetry.ApplicationInsights
+{
+    /// <summary>
+    /// Used to add targeting information to outgoing Application Insights telemetry.
+    /// </summary>
+    public class TargetingTelemetryInitializer : ITelemetryInitializer
+    {
+        private const string TargetingIdKey = $"Microsoft.FeatureManagement.TargetingId";
+
+        /// <summary>
+        /// Creates an instance of the TargetingTelemetryInitializer
+        /// </summary>
+        public TargetingTelemetryInitializer() { }
+
```

Comment:
Is there a reason to define the default constructor?

Reply:
No, removed.